.github/workflows/deploy-to-microk8s.yaml

```
@@ -0,0 +1,122 @@
+name: Validation on MicroK8s
+on:
+  workflow_dispatch:
+    inputs:
+      k8s_version:
+        description: 'Kubernetes version to be used for the AKS cluster. Make sure that the corresponding K8s version is supported by the cloud.'
+        required: false
+      uats_branch:
+        description: 'Branch to run the UATs from e.g. main or track/1.10. By default, this is defined by the dependencies.yaml file.'
+        required: false
+  pull_request:
+  schedule:
+    - cron: "17 02 * * 1"
+
+jobs:
+  deploy-solution:
+    name: Deploy CKF + MLFlow solution to K8s
+    runs-on: ["self-hosted-linux-amd64-jammy-xlarge"]
+
+    strategy:
+      fail-fast: false
+      matrix:
+        bundle:
+          - module: kubeflow-mlflow
+            uats:
+              env: uats-remote
+              extra-args: --filter "not feast"
```

suggestion(blocking): We *should* also test `kubeflow` on its own, right? This would help us more easily identify issues with the `mlflow` integration
```suggestion
          - module: kubeflow
            uats:
              env: kubeflow-remote
          - module: kubeflow-mlflow
            uats:
              env: uats-remote
              extra-args: --filter "not feast"
```

---

We don't have test module for this, compare `modules/kubeflow` with `modules/kubeflow-mlflow`. This is not supported at the moment, and I would not add this as part of the current PR

---

Created https://github.com/canonical/charmed-kubeflow-solutions/issues/86 to track this